orc8r/gateway/python/magma/state/tests/state_replicator_test.py

```diff
@@ -112,6 +114,7 @@ def setUp(self):
         asyncio.set_event_loop(self.loop)
 
         service = MagicMock()
+        service.mconfig.sync_interval = 75
```

Comment:
should this value be verified somewhere?

Reply:
put this in to prevent a type error and was handling all verification in the resolver table tests below but will add coverage here to future proof
`   if mconfig_sync_interval is None or mconfig_sync_interval <= 0: TypeError: '<=' not supported between instances of 'MagicMock' and 'int' `